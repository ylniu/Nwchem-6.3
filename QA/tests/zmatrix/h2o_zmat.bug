 argument  1 = h2o_zmat.nw



============================== echo of input deck ==============================
echo

start h2o_zmat.temp

basis spherical noprint
  * library cc-pvdz
end

scf
  singlet
  rhf
  print low
end

geometry units bohr
  symmetry c1
  O      0.000000000000     0.000000000000    -0.123909374404
  H      0.000000000000     1.429936611037     0.983265845431
  H      0.000000000000    -1.429936611037     0.983265845431
end

task scf energy

geometry units angstrom
 zmatrix
  O
  H 1 R
  H 1 R 2 A
  variables
   R   0.9575091749
   A 104.4784445984
  end
end

task scf energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.1
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = megatron
    program       = nwchem
    date          = Sat Feb 18 17:33:27 2012

    compiled      = Mon_Feb_13_15:04:47_2012
    source        = /software/nwchem/nwchem-trunk
    nwchem branch = Development
    input         = h2o_zmat.nw
    prefix        = h2o_zmat.temp.
    data base     = ./h2o_zmat.temp.db
    status        = startup
    nproc         =        1
    time left     =     -1s



           Memory information
           ------------------

    heap     =   13107201 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428802 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000    -0.22143504
    2 H                    1.0000     0.00000000     1.42993661     0.88574018
    3 H                    1.0000     0.00000000    -1.42993661     0.88574018

      Atomic Mass 
      ----------- 

      O                 15.994910
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       9.1969343804

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95700
    2 Stretch                  1     3                       0.95700
    3 Bend                     2     1     3               104.50000


            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000    -0.11717839
 H                     0.00000000     0.75668992     0.46871355
 H                     0.00000000    -0.75668992     0.46871355

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.80847  |     1.80847
    3 H                |   1 O                |     1.80847  |     1.80847
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.50
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    24
  atoms           =     3
  closed shells   =     5
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./h2o_zmat.temp.movecs
  use symmetry    = F
  symmetry adapt  = F


 Forming initial guess at       0.0s


 Starting SCF solution at       0.1s



       Final RHF  results 
       ------------------ 

         Total SCF energy =    -76.026808177092
      One-electron energy =   -123.154597349507
      Two-electron energy =     37.930854791972
 Nuclear repulsion energy =      9.196934380443

        Time for solution =      0.1s



 Task  times  cpu:        0.1s     wall:        0.2s
Segmentation fault
