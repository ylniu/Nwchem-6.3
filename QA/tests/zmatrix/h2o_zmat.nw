echo

start h2o_zmat.temp

basis spherical noprint
  * library cc-pvdz
end

scf
  singlet
  rhf
  print low
end

geometry units bohr
  symmetry c1
  O      0.000000000000     0.000000000000    -0.123909374404
  H      0.000000000000     1.429936611037     0.983265845431
  H      0.000000000000    -1.429936611037     0.983265845431
end

task scf energy

geometry units angstrom
 zmatrix
  O
  H 1 R
  H 1 R 2 A
  variables
   R   0.9575091749
   A 104.4784445984
  end
end

task scf energy
