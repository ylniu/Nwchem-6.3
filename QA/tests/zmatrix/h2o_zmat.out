 argument  1 = h2o_zmat.nw



============================== echo of input deck ==============================
echo

start h2o_zmat.temp

basis spherical noprint
  * library cc-pvdz
end

scf
  singlet
  rhf
  print low
end

geometry units bohr
  symmetry c1
  O      0.000000000000     0.000000000000    -0.123909374404
  H      0.000000000000     1.429936611037     0.983265845431
  H      0.000000000000    -1.429936611037     0.983265845431
end

task scf energy

geometry units angstrom
 zmatrix
  O
  H 1 R
  H 1 R 2 A
  variables
   R   0.9575091749
   A 104.4784445984
  end
end

task scf energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.0
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2010
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname      = megatron
    program       = /software/nwchem/nwchem-6.0/bin/LINUX64/nwchem
    date          = Sat Feb 18 17:00:48 2012

    compiled      = Mon_Oct_31_14:01:39_2011
    source        = /software/nwchem/nwchem-6.0
    nwchem branch = 6.0
    input         = h2o_zmat.nw
    prefix        = h2o_zmat.temp.
    data base     = ./h2o_zmat.temp.db
    status        = startup
    nproc         =        1
    time left     =     -1s



           Memory information
           ------------------

    heap     =   13107201 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428802 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------


 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in a.u. (scale by  1.000000000 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000    -0.22143504
    2 H                    1.0000     0.00000000     1.42993661     0.88574018
    3 H                    1.0000     0.00000000    -1.42993661     0.88574018

      Atomic Mass 
      ----------- 

      O                 15.994910
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       9.1969343804

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       0.95700
    2 Stretch                  1     3                       0.95700
    3 Bend                     2     1     3               104.50000


            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000    -0.11717839
 H                     0.00000000     0.75668992     0.46871355
 H                     0.00000000    -0.75668992     0.46871355

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |       a.u.
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.80847  |     1.80847
    3 H                |   1 O                |     1.80847  |     1.80847
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.50
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    24
  atoms           =     3
  closed shells   =     5
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = atomic
  output vectors  = ./h2o_zmat.temp.movecs
  use symmetry    = F
  symmetry adapt  = F


 Forming initial guess at       0.0s


 Starting SCF solution at       0.1s



       Final RHF  results 
       ------------------ 

         Total SCF energy =    -76.026808177092
      One-electron energy =   -123.154597349507
      Two-electron energy =     37.930854791972
 Nuclear repulsion energy =      9.196934380443

        Time for solution =      0.1s



 Task  times  cpu:        0.1s     wall:        0.2s


                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 C2V symmetry detected


                         Geometry "geometry" -> "        "
                         ---------------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 O                    8.0000     0.00000000     0.00000000     0.11726921
    2 H                    1.0000     0.75698224     0.00000000    -0.46907685
    3 H                    1.0000    -0.75698224     0.00000000    -0.46907685

      Atomic Mass 
      ----------- 

      O                 15.994910
      H                  1.007825


 Effective nuclear repulsion energy (a.u.)       9.1920946388

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000

      Symmetry information
      --------------------

 Group name             C2v       
 Group number             16
 Group order               4
 No. of unique centers     2

      Symmetry unique atoms

     1    2



                                Z-matrix (user)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

  Constrained variables are marked with a plus sign (+).


      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1+Stretch      R           2     1                       0.95751
    2+Stretch      R           3     1                       0.95751
    3 Bend         A           3     1     2               104.47844


            XYZ format geometry
            -------------------
     3
 geometry
 O                     0.00000000     0.00000000     0.11726921
 H                     0.75698224     0.00000000    -0.46907685
 H                    -0.75698224     0.00000000    -0.46907685

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 H                |   1 O                |     1.80943  |     0.95751
    3 H                |   1 O                |     1.80943  |     0.95751
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          2
 ==============================================================================



 ==============================================================================
                                 internuclear angles
 ------------------------------------------------------------------------------
        center 1       |       center 2       |       center 3       |  degrees
 ------------------------------------------------------------------------------
    2 H                |   1 O                |   3 H                |   104.48
 ------------------------------------------------------------------------------
                            number of included internuclear angles:          1
 ==============================================================================



                                 NWChem SCF Module
                                 -----------------



  ao basis        = "ao basis"
  functions       =    24
  atoms           =     3
  closed shells   =     5
  open shells     =     0
  charge          =   0.00
  wavefunction    = RHF 
  input vectors   = ./h2o_zmat.temp.movecs
  output vectors  = ./h2o_zmat.temp.movecs
  use symmetry    = T
  symmetry adapt  = T


 Forming initial guess at       0.2s


 !! scf_movecs_sym_adapt:   20 vectors were symmetry contaminated

  Symmetry fudging

 !! scf_movecs_sym_adapt:   23 vectors were symmetry contaminated


 Starting SCF solution at       0.2s



       Final RHF  results 
       ------------------ 

         Total SCF energy =    -76.026784861202
      One-electron energy =   -123.145933305141
      Two-electron energy =     37.927053805182
 Nuclear repulsion energy =      9.192094638757

        Time for solution =      0.1s



 Task  times  cpu:        0.1s     wall:        0.1s
 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  564      564     5037     1842     3524        0        0        0     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             3.60e+06 1.47e+06 5.91e+05 0.00e+00 0.00e+00 0.00e+00
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 59712 bytes
MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        18	        14
	current total bytes		         0	         0
	maximum total bytes		   1585384	  22509456
	maximum total K-bytes		      1586	     22510
	maximum total M-bytes		         2	        23


                                NWChem Input Module
                                -------------------





                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
      E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski, T. P. Straatsma,
     M. Valiev, H. J. J. van Dam, D. Wang, E. Apra, T. L. Windus, J. Hammond,
     J. Autschbach, P. Nichols, S. Hirata, M. T. Hackler, Y. Zhao, P.-D. Fan,
      R. J. Harrison, M. Dupuis, D. M. A. Smith, K. Glaesemann, J. Nieplocha,
      V. Tipparaju, M. Krishnan, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
             G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao,
        R. Kendall, J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell,
       D. Bernholdt, P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. Deegan,
        K. Dyall, D. Elwood, E. Glendening, M. Gutowski, A. Hess, J. Jaffe,
        B. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield,
    X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing, G. Sandrone,
       M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. Wong, Z. Zhang.

 Total times  cpu:        0.2s     wall:        0.3s
